app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,83 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do #show all tweets
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :'/tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do #load new tweet form
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do #process new tweet form
+    if logged_in?
+      if params[:content] == ""
+        redirect '/tweets/new'
+      else
+        @tweet = Tweet.create(content: params[:content], user_id: current_user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.